pkg/config/output.go

```diff
@@ -12,11 +12,19 @@ type ModbusOutputData struct {
 	// Type is the type of the data held in the registers. The supported
 	// types are as follows:
 	//
+	//  "b", "bool", "boolean": boolean
+	//  "u16", "uint16":  unsigned 16-bit integer
 	//  "u32", "uint32":  unsigned 32-bit integer
 	//  "u64", "uint64":  unsigned 64-bit integer
+	//  "s16", "int16":   signed 16-bit integer
 	//  "s32", "int32":   signed 32-bit integer
 	//  "s64", "int64":   signed 64-bit integer
 	//  "f32", "float32": 32-bit floating point number
 	//  "f64", "float64": 64-bit floating point number
 	Type string
+
+	// Divisor is the amount to divide by (scale by) to get the actual result.
+	// This could be a float or int.
+	Divisor float64 // TODO: Figure out if this is the same as OutputType.ScalingFactor.
```

Comment:
Would like this to go away if scaling factor works. Need to see it work first, so in the meantime this is here since it works as well.